content/tutorials/spatial_standards_raster/index.Rmd

```diff
@@ -0,0 +1,228 @@
+---
+title: "How to use the GeoTIFF raster file format in R"
+description: "A simple tutorial to demonstrate the use of GeoTIFF files in R"
+author: "Floris Vanderhaeghe"
+date: 2019-11-20
+categories: ["r", "gis"]
+tags: ["gis", "r"]
+output: 
+    md_document:
+        preserve_yaml: true
+        variant: gfm
+---
+
+```{r include = FALSE}
+library(knitr)
+opts_chunk$set(
+  echo = TRUE
+)
+set.seed(123456789)
+unlink("art*")
+```
+
+
+This tutorial uses a few basic functions from the [dplyr](https://dplyr.tidyverse.org) and [raster](https://rspatial.org/raster/pkg) packages.
+While only a few functions are used, you can use the previous hyperlinks to access the tutorials (vignettes) of these packages for more functions and information.
+
+```{r warning=FALSE, message=FALSE}
+options(stringsAsFactors = FALSE)
+library(raster)
+library(tidyverse)
+library(inborutils)
+```
+
+You find bit more background about 'why and what', regarding the open GeoTIFF standard, in [a separate post](../../articles/geospatial_standards/) on this website.
+
+In short, the GeoTIFF format is ideal for exchange, publication, interoperability & durability and to open science in general.
+
+## How to make and use GeoTIFF files (`*.tif`)
```

Comment:
```suggestion
## How to make and use of GeoTIFF files (`*.tif`)
```

Reply:
it is not 'how to make use of' but 'make + use'